infrastructure/variables.tf

```diff
@@ -35,4 +35,18 @@ variable "common_tags" {
 variable "appinsights_instrumentation_key" {
   description = "Instrumentation key of the App Insights instance this webapp should use. Module will create own App Insights resource if this is not provided"
   default     = ""
-}
\ No newline at end of file
+}
+
+/// POSTGRES FLEXIBLE SERVER DB VARIABLES
+variable "pgsql_sku" {
+  description = "The PGSql flexible server instance sku"
+  default     = "GP_Standard_D2s_v3"
+}
+
+variable "pgsql_storage_mb" {
+  description = "Max storage allowed for the PGSql Flexibile instance"
+  type        = number
+  default     = 65536
```

Comment:
65 GB is that we are good with ? I think it might be ok as current one is 50GB

Reply:
shall I change it to 50?